src/vstest.console/CommandLine/CommandLineOptions.cs

```diff
@@ -84,6 +83,7 @@ protected CommandLineOptions()
             this.BatchSize = DefaultBatchSize;
             this.TestStatsEventTimeout = this.DefaultRetrievalTimeout;
             this.FileHelper = new FileHelper();
```

Comment:
check if filehelper is no longer needed remove it

Reply:
Still needed to determine if the path is relative or absolute